geode-redis/src/main/java/org/apache/geode/redis/internal/netty/ExecutionHandlerContext.java

```diff
@@ -264,7 +238,7 @@ private void moveSubscribeToNewEventLoopGroup(ChannelHandlerContext ctx, Command
     if (command.isOfType(SUBSCRIBE)) {
       CountDownLatch latch = new CountDownLatch(0);
```

Comment:
This looks like a bug in that the latch should be initialized with `1`. This is my bug :(.